.github/workflows/rspec.yml

```diff
@@ -0,0 +1,37 @@
+name: RSpec
+on:
+  - push
```

Comment:
You can restrict this to specific branches if needed (e.g. only on `release` or `master`) to avoid the same action running twice for PRs and to avoid the action running for commits before the PR is opened.

I don't really have the context of what we're going for here, so it might be intentional, just commenting FYI.

Reply:
I noticed that it was running the test suite more than necessary. I don't particularly care about handling PRs differently from branch commits so perhaps I can just remove the `pull_request` part?